### PR TITLE
Update list of maintainers and add instructions for contributors

### PR DESCRIPTION
**Pull request by lmontrieux**
This PR adds a new CONTRIBUTING.md file, which provides information
for new contributors on how to get started. This file is adapted from
Skipper's version
(https://github.com/zalando/skipper/blob/master/CONTRIBUTING.md)

This PR also updates the list of contributors.

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+Thanks for your contributions!
+
+### Commit messages
+Your commit messages ideally can answer two questions: what changed and why. The subject line should feature the “what” and the body of the commit should describe the “why”.
+
+When creating a pull request, its comment should reference the corresponding issue id.
+
+**Have fun, and happy hacking!**
+Vyacheslav Stepanov <vyacheslav.stepanov@zalando.de>
+Lionel Montrieux <lionel.montrieux@zalando.de>
```
Comment (adyach) [inline MAINTAINERS]: do you know why the list of maintainers is separated into two ?

Comment (lmontrieux) [inline MAINTAINERS]: I must have accidentally pasted another file here. Fixed in 08f671c

**Comment by adyach** [comment]
👍

File: MAINTAINERS [hunk 2]
```diff
@@ -1,2 +1,7 @@
+Andrey Dyachkov <andrey.dyachkov@zalando.de>
 Dmitry Sorokin <dmitriy.sorokin@zalando.de>
-
+Ricardo De Cillo <ricardo.de.cillo@zalando.de>
+Valentine Gogichashvili <valentine.gogichashvili@zalando.de>
+Vyacheslav Stepanov <vyacheslav.stepanov@zalando.de>
+Lionel Montrieux <lionel.montrieux@zalando.de>
+Robert Garrett <robert.garrett@zalando.de>
```
Comment (antban) [inline MAINTAINERS]: @lmontrieux Can you please remove some file parts or close pr if it is not needed?

Comment (lmontrieux) [reply MAINTAINERS]: oooh, this is an old one :). Yes, I'll have a look and close the PR if it's not needed anymore

**Comment by lmontrieux** [comment]
@antban now I've fixed it. we should merge this to conform to the open source team requirements

File: MAINTAINERS [hunk 3]
```diff
@@ -1,2 +1,7 @@
+Andrey Dyachkov <andrey.dyachkov@zalando.de>
 Dmitry Sorokin <dmitriy.sorokin@zalando.de>
-
+Ricardo De Cillo <ricardo.de.cillo@zalando.de>
+Vyacheslav Stepanov <vyacheslav.stepanov@zalando.de>
+Lionel Montrieux <lionel.montrieux@zalando.de>
+Suyash Gard <suyash.garg@zalando.de>
```
Comment (antban) [inline MAINTAINERS]: @lmontrieux typo here ))

Comment (lmontrieux) [reply MAINTAINERS]: oops. Fixed it.

**Comment by lmontrieux** [comment]
👍

**Pull request merged**